client/lib/abtest/index.js

```diff
@@ -87,6 +88,13 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
+	const languageSlugs = map( config( 'languages' ), 'langSlug' );
+	const localeTargets = [ 'not-en', 'any' ].concat( languageSlugs );
+
+	if ( testConfig.localeTargets && ! includes( localeTargets, testConfig.localeTargets ) ) {
+		throw new Error( 'localeTargets can by "any", "not-en" or any single locale slug.' );
```

Comment:
Plural variable vs singular value confuses me here. I think I get why `localeTargets` is plural. In case of `any` or `not-en`? FWIW, I think you can get away with singular covering those (ie, `localeTarget`) which feels more consistent to me.